<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBuilderScriptCreator</class>
 <widget class="QWidget" name="DialogBuilderScriptCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Builder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QGroupBox" name="argsGroupBox">
     <property name="title">
      <string>Script Arguments</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="numArgsLabel">
          <property name="text">
           <string>Number of Arguments</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numArgsSpinBox">
          <property name="maximum">
           <number>15</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="promptUserCheckBox">
        <property name="text">
         <string>Prompt User to Input Values</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Default Value</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Filter</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="displayNameLabel">
       <property name="text">
        <string>Display Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="displayNameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="displayIconLabel">
       <property name="text">
        <string>Display Icon</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="displayIconLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="browseIconPushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="buttonColorLabel">
       <property name="text">
        <string>Button Color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="UtQtColorButton" name="colorPushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="linkLabel">
       <property name="text">
        <string>Link</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="linkLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="scriptNameLabel">
       <property name="text">
        <string>Script Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="scriptNameLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="contextLabel">
       <property name="text">
        <string>Context</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="contextComboBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="platformTypeLabel">
       <property name="text">
        <string>Platform Type</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="platformTypeLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="clampPlatformSelectionCheckbox">
     <property name="text">
      <string>Clamp Platform Selection</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtColorButton</class>
   <extends>QPushButton</extends>
   <header>UtQtColorButton.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>displayNameLineEdit</tabstop>
  <tabstop>displayIconLineEdit</tabstop>
  <tabstop>browseIconPushButton</tabstop>
  <tabstop>colorPushButton</tabstop>
  <tabstop>scriptNameLineEdit</tabstop>
  <tabstop>contextComboBox</tabstop>
  <tabstop>platformTypeLineEdit</tabstop>
  <tabstop>numArgsSpinBox</tabstop>
  <tabstop>promptUserCheckBox</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
